resources/authgear/templates/en/web/authflowv2/enter_password.html

```diff
@@ -53,5 +53,94 @@ <h1 class="screen-title">Enter Password</h1>
     By registering, you agree to the <a href="#">Terms of Service</a> and
     <a href="#">Privacy Policy</a>.
   </p>
+
+  <!-- OTP Input -->
+  <div data-controller="otp-input">
+    <div class="otp-input h-11">
+      <input
+        data-otp-input-target="input"
+        type="text"
+        inputmode="numeric"
+        autocomplete="one-time-code"
+        pattern="\d*"
+        maxlength="6"
+        autofocus=""
+        required
+      />
+      <div data-otp-input-target="textContainer" class="text-container"></div>
+    </div>
+    <form method="post" novalidate>
+      {{ $.CSRFField }}
+      <button
+        class="hidden"
+        type="submit"
+        name="x_action"
+        value="submit"
+        data-otp-input-target="submit"
+      ></button>
+    </form>
+    <form method="post" novalidate>
+      <button
```

Comment:
After to confirming with designer, this button behaves more line an inline element. Should I just use span with role button?

Reply:
Use `button` and style it with a different style.